app/models.py

```diff
@@ -607,9 +608,6 @@ def __repr__(self):
         return '<{} has service permission: {}>'.format(self.service_id, self.permission)
```

Comment:
while we are here, can we just convert this to an f-string?

Reply:
Happily, but I am the guy people like to cite as somebody who introduces unrelated changes . . .